backend/source/archive/models/google_takeout.py

```diff
@@ -202,3 +204,45 @@ def extract_history_entries(self, json_files: Iterable[Path], schema: str,
                     except:
                         logging.exception(f"Could not parse entry: {entry}")
                         raise
+
+    def extract_fit_history(self) -> Generator[Entry, None, None]:
+        json_files = list((self.extracted_files_path / 'Takeout/Fit/All sessions/').glob('*.json'))
+        logger.info(f'Processing fit history in "{self.entry_source}". '
+                    f'{len(json_files)} files found.')
+
+        for json_file in json_files:
+            logger.info(f'Processing fit history entries in {str(json_file)}')
+            with json_file.open(encoding='utf-8') as json_file_handle:
+                json_entry = json.load(json_file_handle)
+
+            try:
+                time = pytz.utc.localize(datetime.strptime(json_entry['startTime'], '%Y-%m-%dT%H:%M:%S.%fZ'))
+            except ValueError:
+                time = json_to_datetime(json_entry['startTime'])
+
+            try:
+                activity = json_entry['fitnessActivity']
+                duration_sec = json_entry['duration'][:-1] # ignore final "s" for second
```

Comment:
Is it always measured in seconds? I'd add an `assert` to make sure the string indeed matches `^[0-9]+s$`.